### DISCUSSION TITLE
EQ5 MyOpenMath

### DISCUSSION THREAD

**Discussion by tdegeorge** [TeamBasedInquiryLearning/precalculus#342] [General]
Reviewer: @AbbyANoble 

UPDATED homework set in MOM

Question 1 - 3: Solve by factoring (already set equal to 0)
Questions 4 - 8: Mixture of solve by factoring (GCF, a=1, a not equal to 1, etc.)
Questions 9 - 13: solve using square root property (including a binomial squared)
Questions 14 - 19: Solve by completing the square. Problems 14 - 18 are problems with a=1 (with the last one saying it has complex roots), problems 18 and 19 are with a not equal to 1
Questions 20 - 23: Basic identifying a, b, and c from an equation & scaffolded.
Questions 24 - 26: Solve by quadratic formula (asks for both exact and approximate answers)
Questions 27 - 32: Mixture of solving by quadratic formula
Questions 33 - 34: Mixture  - does not specify which method to use
Question 35: Solving an equation with x^4.

**Comment by AbbyANoble** [comment]
Some of these are static, which may or may not be something we want to use. I generally don't use them because I allow multiple attempts. They are the ones marked with this symbol: 
<img width="29" alt="Screenshot 2024-07-01 at 12 13 44 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/20615e9a-e7c1-4fda-b410-901690dde3e5">

I can easily take those out for my personal copy though.

**Reply by tdegeorge** [reply]
Took out the static ones.

**Comment by AbbyANoble** [comment]
Also, this assignment has a lot more problems than I've been including! Mine have been around 20 questions I think. Maybe we need to talk about how large we want to make these at our meeting tomorrow?

**Reply by tdegeorge** [reply]
Shortened it and updated the types of questions in the original post.

**Comment by AbbyANoble** [comment]
Might be more factoring ones than needed, especially since the first 2-4 have multiple in the one problem.

**Reply by AbbyANoble** [reply]
Maybe take some of the ones where it doesn't say to factor that you were going to edit and throw them in the mixed section where they aren't told what to do.

**Comment by AbbyANoble** [comment]
25 has a typo in it:

<img width="764" alt="Screenshot 2024-07-01 at 12 26 48 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/f69a34ac-b2fa-44d2-8a29-88dd6d63bd71">

**Reply by tdegeorge** [reply]
I just removed that question. I was afraid that if I edit it, then it resets the stats. Figured it wasn't worth it...

**Comment by AbbyANoble** [comment]
About editing them, you can click on the 3 dots at the end of the problem and click Edit Personal Copy.

The only thing about editing them is that since it is new it resets the stats that let you know how many times it's been used and how long it took. You can see it for the original, but yours won't have it. (Well it will start collecting new.) Not sure we care, but that is something to consider. A lot of times when I find one I want to edit, I just try to find a different one that is similar and use it instead. haha.

<img width="1034" alt="Screenshot 2024-07-01 at 12 28 32 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/85640990/5ffc8626-0006-4adf-916c-ca4ccec4b055">
